internal/integration/script_test.go

```diff
@@ -310,6 +317,31 @@ func (t *liteTest) cmdExec(ts *testscript.TestScript, _ bool, args []string) {
 	cmdExec(ts, args, ts.Value(keyDB).(*sql.DB))
 }
 
+func (t *liteTest) cmdCLI(ts *testscript.TestScript, _ bool, args []string) {
+	for i, arg := range args {
+		args[i] = strings.ReplaceAll(arg, "URL", t.dsn())
+	}
+	l := len(args)
+	switch {
+	// If command was run with a unix redirect-like suffix.
+	case l > 1 && args[l-2] == ">":
+		ts.Check(func() error {
+			wd := ts.Getenv("WORK")
+			path := filepath.Join(wd, args[l-1])
+			out, err := os.Create(path)
+			if err != nil {
+				return err
+			}
+			defer out.Close()
+			cmd := exec.Command(ts.Getenv("cli"), args[0:l-2]...)
+			cmd.Stdout = out
+			return cmd.Run()
+		}())
```

Comment:
why it's better than:
```suggestion
		wd := ts.Getenv("WORK")
		path := filepath.Join(wd, args[l-1])
		out, err := os.Create(path)
		ts.Check(err)
		defer out.Close()
		cmd := exec.Command(ts.Getenv("cli"), args[0:l-2]...)
		cmd.Stdout = out
		ts.Check(cmd.Run())
```